src/main/java/mekanism/common/CommonPlayerTickHandler.java

```diff
@@ -179,21 +182,24 @@ public static boolean handleJetpackMotion(Player player, JetpackMode mode, Boole
         return true;
     }
 
-    private static boolean isJetpackOn(Player player, ItemStack chest) {
+    private static ItemStack getJetpackIfOn(ItemStack chest, Player player) {
+        if (chest.isEmpty() && Mekanism.hooks.CuriosLoaded) {
+            chest = CuriosIntegration.findFirstCurio(player, s -> s.getItem() instanceof ItemJetpack).orElse(ItemStack.EMPTY);
```

Comment:
I am unsure whether or not we should only be doing this if the chest stack is empty or should it be when it isn't a jetpack or something?  Either way I know the findFirstCurio is to handle there potentially being multiple jetpacks in the curio slots, but do we want to have better handling if say two jetpacks are equipped but only one has hydrogen that we then find the one that has hydrogen?

Reply:
That's a fair point. It should get the first jetpack with hydrogen. That will probably need fixing in the key binding to add that "check" as well.